doc/locale/ja/LC_MESSAGES/reference/commands/status.po

```diff
@@ -232,3 +232,20 @@ msgstr "このGroongaプロセスのバージョンです。"
 
 msgid "``5.0.7``"
 msgstr ""
+
+msgid "``features``"
+msgstr ""
+
+msgid ""
+"The list of features that Groonga has and whether they are enabled or not."
```

Comment:
```suggestion
"The list of Groonga's features and status(enabled or distable)"
```
Above suggestion  for simple description. 
If detailed description is preferred as follows;

"The list shows features of Groonga and whether they are enabled or disabled."

When you used " enabled" as a adjective to explain status it would be better to say " enabled or disabled".

Reply:
もし、リストに有効なものだけが出るというのであれば、言い方を変えたほうがいいので教えてください。

Reply:
ありがとうございます。リストには有効なものと無効なものどちらも出るので、この案でいかせてもらいます